test/index.js

```diff
@@ -96,6 +97,48 @@ describe('KISSmetrics', function () {
     });
   });
 
+  describe.only('.completedOrder()', function(){
+    it('should send a completed order call correctly', function(done){
+      settings.prefixProperties = true;
+      var json = test.fixture('track-ecommerce');
+
+      test
+        .requests(3)
+        .request(0)
```

Comment:
ah yup, that's needs to be documented but here's a full example of testing 3 requests:
https://github.com/segmentio/integration-tester/blob/master/test/assertion.js#L594-L641